lib/presentation/components/post_card/contents_only_post_card.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:gap/gap.dart';
+import 'package:social_mobile/domain/post/post.dart';
+import 'package:social_mobile/utils/helpers/date_time_format.dart';
+
+class ContentsOnlyPostCard extends StatelessWidget {
+  const ContentsOnlyPostCard({super.key, required this.post});
+
+  final Post post;
+
+  @override
+  Widget build(BuildContext context) {
+    final textStyle = Theme.of(context).textTheme;
+
+    return Card(
+      elevation: 0,
+      margin: const EdgeInsets.fromLTRB(16, 0, 16, 0),
```

Comment:
わかりやすいようにfactoryメソッドを使いましょう！
以下は横に16ずつという意味です
```suggestion
      margin: const EdgeInsets.symmetric(horizontal: 16),
```